### DISCUSSION TITLE
jsmind是否允许禁止缩放

### DISCUSSION THREAD

**Discussion by avrinfly** [hizzgdev/jsmind#572] [General]
要求画布不能缩放，但我看option里有快捷键操作，理论上crtl+鼠标滚轮，应该也是快捷键，但现在无法禁止

**Comment by avrinfly** [comment]
> 要求画布不能缩放，但我看option里有快捷键操作，理论上crtl+鼠标滚轮，应该也是快捷键，但现在无法禁止

我把zoom的最大缩放比例和最小缩放比例都设为默认的1，就可以禁止了，但还是建议在option里有字段来控制

**Reply by hizzgdev** [reply]
好的，后面我考虑一下如何定义鼠标操作。谢谢
